### PR TITLE
[ADD] sale_order_crm_lead_creation

### PR DESCRIPTION
**Pull request by JorgeQuinteros**

### CODE REVIEW

File: sale_order_crm_lead_creation/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+###############################################################################
+#
+#    SDi Soluciones Digitales
```
Comment (JorgeQuinteros) [inline sale_order_crm_lead_creation/__manifest__.py]:
```suggestion
#    SDi Digital Group
```

File: sale_order_crm_lead_creation/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+{
+    'name': 'Sale Order CRM Lead',
+    'summary': 'Create Opportuniy from sale order with default values',
+    'author': 'Jorge Luis Quinteros, SDi Soluciones Digitales',
```
Comment (JorgeQuinteros) [inline sale_order_crm_lead_creation/__manifest__.py]:
```suggestion
    'author': 'Jorge Luis Quinteros, SDi Digital Group',
```

File: sale_order_crm_lead_creation/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+###############################################################################
+#
+#    SDi Digital Group
+#    Copyright (C) 2021-Today SDi Soluciones Digitales <www.sdi.es>
```
Comment (JorgeQuinteros) [inline sale_order_crm_lead_creation/__manifest__.py]: ``
```suggestion
#    Copyright (C) 2021-Today SDi Digital Group <www.sdi.es>
```

File: sale_order_crm_lead_creation/views/sale_order_views.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    <field name="arch" type="xml">
+        <field name="opportunity_id" position="attributes">
+            <attribute name="context">{
+                'default_type': 'opportunity',
+                'default_partner_id': partner_id,
+                'default_planned_revenue': amount_total,
+            }</attribute>
+        </field>
+    </field>
+</record>
```
Comment (valentincastravete) [inline sale_order_crm_lead_creation/views/sale_order_views.xml]:
```suggestion
    <record id="view_order_form" model="ir.ui.view">
        <field name="model">sale.order</field>
        <field name="inherit_id" ref="sale.view_order_form"/>
        <field name="arch" type="xml">
            <field name="opportunity_id" position="attributes">
                <attribute name="context">{
                    'default_type': 'opportunity',
                    'default_partner_id': partner_id,
                    'default_planned_revenue': amount_total,
                }</attribute>
            </field>
        </field>
    </record>
```

**Review by valentincastravete — APPROVED** [review]
LGTM!

**Review by valentincastravete — APPROVED** [review]
LGTM!

**Pull request merged**